src/php/generator.php

```diff
@@ -0,0 +1,124 @@
+<?PHP
+
+    $design = $_POST['design'];
+    $first_name = $_POST['first_name'];
+    $last_name = $_POST['last_name'];
+    $home = $_POST['home'];
+    $city = $_POST['city'];
+    $country = $_POST['country'];
+    $postal_code = $_POST['postal_code'];
+    $type_phone = $_POST['type_phone'];
+    $number_phone = $_POST['number_phone'];
+    $email = $_POST['email'];
+    $date_job_start = $_POST['date_job_start'];
+    $date_job_end = $_POST['date_job_end'];
+    @$current_job = $_POST['current_job'];
+    $job = $_POST['job'];
+    $employer_name = $_POST['employer_name'];
+    $employer_city = $_POST['employer_city'];
+    $employer_country = $_POST['employer_country'];
+    $date_education_start = $_POST['date_education_start'];
+    $date_education_end = $_POST['date_education_end'];
+    @$current_education = $_POST['current_education'];
```

Comment:
Supongo que será así

```suggestion
    $current_education = $_POST['current_education'];
```